shell/platform/windows/flutter_window.h

```diff
@@ -168,6 +169,8 @@ class FlutterWindow : public Window, public WindowBindingHandler {
 
   // The cursor rect set by Flutter.
   RECT cursor_rect_;
+
+  FML_DISALLOW_COPY_AND_ASSIGN(FlutterWindow);
```

Comment:
Should `Window` in `window.h` get the same treatment?

Reply:
Good question! Window is an abstract class so already not copyable/movable.

Reply:
Actually, I'm going to pass on that for now as it would involve adding a bunch of default constructors since we're explicitly deleting other ctors.